dbt/adapters/sqlserver/connections.py

```diff
@@ -68,7 +68,7 @@ def _connection_keys(self):
         
     
         return 'server', 'database', 'schema', 'port', 'UID', \
-                 'authentication', 'encrypt'
+               'client_id','authentication', 'encrypt'
```

Comment:
ClientId was not displayed in `dbt debug` window for `Auth=ServicePrincipal`
![image](https://user-images.githubusercontent.com/67547199/94100724-ecf98800-fe4b-11ea-9e3d-1b736383841f.png)